notmuch/mutt_notmuch.c

```diff
@@ -2173,7 +2173,6 @@ static int nm_mbox_open(struct Mailbox *m, struct Context *ctx)
   m->mtime.tv_sec = time(NULL);
   m->mtime.tv_nsec = 0;
 
-  mx_update_context(ctx, m->msg_count);
```

Comment:
Removing this `mx_update_context(...)` call breaks the progress bar `notmuch`. This appears to be only back-end whose progress bar is broken by removing `mx_update_context(...)` so it must be an implementation issue.